sklearn/metrics/classification.py

```diff
@@ -2168,7 +2166,11 @@ def log_loss(y_true, y_pred, eps=1e-15, normalize=True, sample_weight=None,
                              'labels: {0}'.format(lb.classes_))
 
     # Renormalize
-    y_pred /= y_pred.sum(axis=1)[:, np.newaxis]
+    y_pred = y_pred / y_pred.sum(axis=1)[:, np.newaxis]
```

Comment:
You don't need to change this line.

Reply:
This relates to this issue https://github.com/numpy/numpy/issues/10565